mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -5258,292 +5258,292 @@ fn test_no_login() {
-        "mmrpc": "2.0",
-        "method": "gui_storage::activate_coins",
-        "params": {
-            "account_id": {
-                "type": "iguana"
-            },
-            "tickers": ["RICK", "MORTY", "KMD"],
-        },
-    })))
-    .unwrap();
-    assert!(resp.0.is_success(), "!gui_storage::activate_coins: {}", resp.1);
-
-    let resp = block_on(mm.rpc(&json!({
-        "userpass": mm.userpass,
-        "mmrpc": "2.0",
-        "method": "gui_storage::activate_coins",
-        "params": {
-            "account_id": {
-                "type": "hw",
-                "device_pubkey": "1549128bbfb33b997949b4105b6a6371c998e212"
-            },
-            "tickers": ["KMD", "MORTY", "BCH"],
-        },
-    })))
-    .unwrap();
-    assert!(resp.0.is_success(), "!gui_storage::activate_coins: {}", resp.1);
-
-    let resp = block_on(mm.rpc(&json!({
-        "userpass": mm.userpass,
-        "mmrpc": "2.0",
-        "method": "gui_storage::deactivate_coins",
-        "params": {
-            "account_id": {
-                "type": "hw",
-                "device_pubkey": "1549128bbfb33b997949b4105b6a6371c998e212"
-            },
-            "tickers": ["BTC", "MORTY"],
-        },
-    })))
-    .unwrap();
-    assert!(resp.0.is_success(), "!gui_storage::deactivate_coins: {}", resp.1);
-
-    let resp = block_on(mm.rpc(&json!({
-        "userpass": mm.userpass,
-        "mmrpc": "2.0",
-        "method": "gui_storage::get_enabled_account",
-    })))
-    .unwrap();
-    assert!(resp.0.is_success(), "!gui_storage::get_enabled_account: {}", resp.1);
-    let actual: RpcV2Response<gui_storage::AccountWithCoins> = json::from_str(&resp.1).unwrap();
-    let expected = gui_storage::AccountWithCoins {
-        account_id: gui_storage::AccountId::Iguana,
-        name: "My Iguana wallet".to_string(),
-        description: String::new(),
-        balance_usd: BigDecimal::from(0i32),
-        coins: vec!["RICK".to_string(), "MORTY".to_string(), "KMD".to_string()]
-            .into_iter()
-            .collect(),
-    };
-    assert_eq!(actual.result, expected);
-
-    let resp = block_on(mm.rpc(&json!({
-        "userpass": mm.userpass,
-        "mmrpc": "2.0",
-        "method": "gui_storage::get_account_coins",
-        "params": {
-            "account_id": {
-                "type": "hw",
-                "device_pubkey": "1549128bbfb33b997949b4105b6a6371c998e212"
-            }
-        }
-    })))
-    .unwrap();
-    assert!(resp.0.is_success(), "!gui_storage::get_enabled_account: {}", resp.1);
-    let actual: RpcV2Response<gui_storage::AccountCoins> = json::from_str(&resp.1).unwrap();
-    let expected = gui_storage::AccountCoins {
-        account_id: gui_storage::AccountId::HW {
-            device_pubkey: "1549128bbfb33b997949b4105b6a6371c998e212".to_string(),
-        },
-        coins: vec!["KMD".to_string(), "BCH".to_string()].into_iter().collect(),
-    };
-    assert_eq!(actual.result, expected);
-}
+// #[test]
```

Comment:
How about instead of commenting out this code we move it to a separate mod that we don't declare it in the parent module so it's not included in the module tree at all.